pmd-core/src/main/java/net/sourceforge/pmd/cpd/renderer/CPDRendererAdapter.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.cpd.renderer;
+
+import java.io.IOException;
+import java.io.Writer;
+
+import net.sourceforge.pmd.cpd.CPDReport;
+
+public class CPDRendererAdapter implements CPDReportRenderer {
```

Comment:
I'll mark this as internal API, since this is just a utility for us to avoid changing all renderers now (which we eventually have to do anyway).